content/software-security/learning-labs/ll202507.md

```diff
@@ -91,11 +91,9 @@ The demo showcases how Chainguard's hardened PyTorch image provides the same fun
 
 ## Resources
 
-- [Event page](https://goldcast.ondemand.goldcast.io/on-demand/3ec17730-bff5-40b3-b5d9-23c90ac26828)
```

Comment:
I would keep the event page

Reply:
It has no relevant content that is not on this page and actually will probably be deleted ... so I think we should not have it here .. if we really want .. it should be the last link .. not the first one

Reply:
it only makes sense to have it when it is the registration page before the event and if we publish the learning labs page before the evnet .. which happened last time .. but now it makes no sense